engine/src/main/java/org/terasology/logic/characters/interactions/InteractionSystem.java

```diff
@@ -61,6 +62,36 @@ public void onActivate(ActivateEvent event, EntityRef target) {
 
     }
 
+    /**
+     * To consume the activation event in the edit mode if the target entity has an EditScreenComponent with no screen defined.
+     * @param event
```

Comment:
Maybe some javadoc for what's expected in the `@param`s beyond their name? Like a typical use case.